src/shared/cfg.py

```diff
@@ -159,6 +163,21 @@ def DEPLOYED_ENV(self):
             return "qa"
         return "dev"
 
+    @property
+    def DEPLOYED_BY(self):
+        """
+        DEPLOYED_BY
+        """
+        return self("DEPLOYED_BY", f"{self.USER}@{self.HOSTNAME}")
+
+    @property
+    @lru_cache()
```

Comment:
Why do we want to cache this?

Reply:
so that the time reported by WHEN is always the same (it is accessed many times across many things)